youtube_dl/extractor/hearthisat.py

```diff
@@ -91,7 +91,7 @@ def _real_extract(self, url):
 
         formats = []
         mp3_url = self._search_regex(
-            r'(?s)<a class="player-link"\s+(?:[a-zA-Z0-9_:-]+="[^"]+"\s+)*?data-mp3="([^"]+)"',
+            r'(?s)<a class="player-link"\s+(?:[a-zA-Z0-9_:-]+="[^"]+"\s+)*?data-mp3="(?!https://hearthis.at/hearthisat/untitled/stream.mp3)([^"]+)"',
```

Comment:
Needs escaping.

Reply:
thx @rautamiekka for the hint!
I think I fixed it with a 2nd commit – https://github.com/ytdl-org/youtube-dl/pull/29453/commits/d323192f950a9b6adc089b6dbb6a79dc0f89ca0d

Reply:
Ya :)